src/Admin/AbstractAdmin.php

```diff
@@ -2441,6 +2442,13 @@ public function transChoice($id, $count, array $parameters = [], $domain = null,
             __METHOD__
         ), E_USER_DEPRECATED);
 
+        if (!$this->translator instanceof LegacyTranslatorInterface) {
+            throw new \RuntimeException(sprintf(
+                'AbstractAdmin::transChoice is only supported for translators implementing "%s"',
+                LegacyTranslatorInterface::class
+            ));
+        }
+
         $domain = $domain ?: $this->getTranslationDomain();
 
         return $this->translator->transChoice($id, $count, $parameters, $domain, $locale);
```

Comment:
```suggestion
        
        if (method_exists($this->translator, 'transChoice')) {
            return $this->translator->transChoice($id, $count, $parameters, $domain, $locale);
        } else {
            return $this->trans($id, ['%count%' => $count] + $parameters, $domain, $locale);
        }
```

Based on: https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Translation/IdentityTranslator.php#L57-L71

Reply:
makes sense I think :+1: changed it